test/support/hipchat_client.ex

```diff
@@ -47,6 +58,15 @@ defmodule Cog.Test.Support.HipChatClient do
     end
   end
 
+  defp wait_for_join(full) do
+    receive do
+      {:stanza, %Stanza.Message{body: "", from: %Romeo.JID{full: ^full}}} ->
+        :ok
+    after
+        @default_timeout -> raise "Error: Timeout waiting for bot to join room"
```

Comment:
Let's put the body of the clause on an indented line by itself (per usual Elixir formatting), and add the name of the room we didn't join to the exception message.